docker-compose.yml

```diff
@@ -54,55 +62,66 @@ services:
       CHARON_DATA_DIR: /charon/bootnode
       CHARON_P2P_BOOTNODES: ""
       CHARON_P2P_EXTERNAL_HOSTNAME: bootnode
-
-  vc0-lh:
+    volumes:
+      - ./:/charon/
+  #             _ _     _       _                 
+  # __   ____ _| (_) __| | __ _| |_ ___  _ __ ___ 
+  # \ \ / / _` | | |/ _` |/ _` | __/ _ \| '__/ __|
+  #  \ V / (_| | | | (_| | (_| | || (_) | |  \__ \
+  #   \_/ \__,_|_|_|\__,_|\__,_|\__\___/|_|  |___/
+  vc0:
     build: lighthouse
     networks: [cluster]
     depends_on: [node0]
+    env_file: .env
     environment:
       NODE: node0
     volumes:
-      - .:/charon
+      - ./lighthouse:/opt/charon/lighthouse
+      - .charon/cluster/node0:/opt/charon/keys
 
-  vc1-teku:
-    image: consensys/teku:latest
+  vc1:
+    image: consensys/teku:22.5.0
```

Comment:
why not latest? specific versions would have an extra overhead to update at the time of next release and if ignored to update can lead to consequences probably

Reply:
See my comment in discord about docker latest tags not actually being the latest image if you don't query the hub again to update what image `latest` now points to. Keeping the version specific keeps us in control rather than images staying nearly the same since the person first checked out and ran this code.